src/main/kotlin/org/rust/ide/core/overrideImplement/generateUtil.kt

```diff
@@ -0,0 +1,112 @@
+package org.rust.ide.core.overrideImplement
```

Comment:
Personally I prefer `camelCase` for files without *main* class, but in intellij-rust codebase we use mostly `UpperCase` similar to class files.